timetracker/timetracker/urls.py

```diff
@@ -16,12 +16,22 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.http import HttpResponse
+import random
```

Comment:
Random was imported because my husband messed with my coding and put love dice in it -.-

Sent from my Sony Xperia™ smartphone on the Telstra™ Mobile network

---- Curtis Maloney wrote ----

>In timetracker/timetracker/urls.py:
>
>> @@ -16,12 +16,22 @@ > from django.conf.urls import include, url > from django.contrib import admin > from django.http import HttpResponse > +import random 
>
>Why was this imported? You've not used it anywhere.
>
>—
>Reply to this email directly or view it on GitHub.￼
>